src/kbmod/analysis_utils.py

```diff
@@ -1028,28 +1026,23 @@ def _cluster_results(self, results, x_size, y_size, v_lim, ang_lim, mjd_times, c
         """
         This function clusters results and selects the highest-likelihood
         trajectory from a given cluster.
-        INPUT-
-            results : kbmod results
-                A list of kbmod trajectory results such as are stored in
-                keep['results'].
-            x_size : list
-                The width of the images used in the kbmod stack, such as are
-                stored in image_params['x_size'].
-            y_size : list
-                The height of the images used in the kbmod stack, such as are
-                stored in image_params['y_size'].
-            v_lim : list
-                The velocity limits of the search, such as are stored in
-                image_params['v_lim'].
-            ang_lim : list
-                The angle limits of the search, such as are stored in
-                image_params['ang_lim']
-            cluster_args : dictionary
-                Arguments to pass to dbscan or OPTICS.
-        OUTPUT-
-            top_vals : numpy array
-                An array of the indices for the best trajectories of each
-                individual cluster.
+
+        Arguments:
+            results (list): A list of kbmod trajectory results such as are
+                stored in keep['results'].
+            x_size (int):  The width of the images used in the kbmod stack,
+                such as are stored in image_params['x_size'].
+            y_size (int):  The height of the images used in the kbmod stack,
+                such as are stored in image_params['y_size'].
+            v_lim (list):  The velocity limits of the search, such as are
```

Comment:
It's weird and not immediately obvious why some of the args are ints and some lists (and it's different from the original docstring).

Reply:
I expanded the comments to explain why v_lim and ang_lim are lists and what they contain. The x_size and y_size were incorrectly described as lists in the former comment.